packages/dds/tree/src/simple-tree/proxies.ts

```diff
@@ -55,27 +56,28 @@ export function isTreeNode(candidate: unknown): candidate is TreeNode | Unhydrat
  * Retrieve the associated proxy for the given field.
  * */
 export function getProxyForField(field: FlexTreeField): TreeNode | TreeValue | undefined {
+	function tryToUnboxLeaves(
+		flexField:
+			| FlexTreeTypedField<FlexFieldSchema<typeof FieldKinds.required>>
+			| FlexTreeTypedField<FlexFieldSchema<typeof FieldKinds.optional>>,
```

Comment:
This is a less redundant way to specify the same type.
```suggestion
		flexField: FlexTreeTypedField<FlexFieldSchema<typeof FieldKinds.required | typeof FieldKinds.optional>>,
```

Reply:
Thanks! Somehow I had confused myself into thinking that meant something else, but you're right.